library/src/main/java/com/readystatesoftware/chuck/api/ChuckInterceptor.java

```diff
@@ -73,6 +80,14 @@ public ChuckInterceptor maxContentLength(long max) {
         return this;
     }
 
+    public ChuckInterceptor redactHeader(String name) {
+        Set<String> newHeadersToRedact = new TreeSet<>(String.CASE_INSENSITIVE_ORDER);
+        newHeadersToRedact.addAll(headersToRedact);
+        newHeadersToRedact.add(name);
+        headersToRedact = newHeadersToRedact;
+        return this;
```

Comment:
Why don't you just `headersToRedact.add(name)`?

Reply:
Done

Reply:
Add it to **library-no-op** too please :wink: